test_s3.py

```diff
@@ -1,29 +1,70 @@
 import boto3
+import sys
+import logging
+import argparse
+import yaml
 
-# Create S3
+bucket_name = 'test-bucket-boto-mzol'
+s3resource = boto3.resource('s3')
+bucket = s3resource.Bucket(bucket_name)
 s3 = boto3.client('s3')
 
-bucket_name = 'test-bucket-boto-mzol'
-print('Creating new bucket with name: Test-Bucket-boto-mzol')
-s3.create_bucket(Bucket=bucket_name)
+"""Create S3"""
+def CreateS3(bucket_name,logger):
+    logger.info("Creating new bucket with name: {}".format(bucket_name));
+    s3.create_bucket(Bucket=bucket_name)
 
-# upload file
-filename = 'text'
-print('Uploading some data to {} file: {}'.format(bucket_name, filename))
-s3.upload_file(filename, bucket_name, filename)
+"""upload file"""
+def UploadFiletoS3(filename, bucket_name,logger):
+    logger.info("Uploading some data to {} file: {}".format(bucket_name, filename))
+    s3.upload_file(filename, bucket_name, filename)
 
-# read file from S3
-s3resource = boto3.resource('s3')
-bucket = s3resource.Bucket(bucket_name)
-obj = bucket.Object(filename)
-print('Object body: {}'.format(obj.get()['Body'].read()))
+"""read file from S3"""
+def ReadFileS3(filename, bucket_name,logger):
+    s3resource = boto3.resource('s3')
+    bucket = s3resource.Bucket(bucket_name)
+    obj = bucket.Object(filename)
+    logger.info('Object body: {}'.format(obj.get()['Body'].read()))
+
+"""remove S3"""
+def DeleteS3(bucket_name,logger):
+    logger.info('Deleting all objects in bucket {}'.format(bucket_name))
+    bucket.objects.all().delete()
+    logger.info('Deleting the bucket')
+    bucket.delete()
+    logger.info('The bucket {} has benn deleted'.format(bucket_name))
+
+"""init logging"""
+def init_logging(level,type):
+    obj = {
+        "STD_OUT": sys.stdout
+    }
+    logger = logging.getLogger("MyLog-S3")
+    ch = logging.StreamHandler(obj[str(type).upper()])
+    logger.setLevel(logging.getLevelName(level))
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    ch.setFormatter(formatter)
+    logger.addHandler(ch)
+    return logger
 
-# Delete all objects in S3
-print('\nDeleting all objects in bucket {}.'.format(bucket_name))
-bucket.objects.all().delete()
+"""init parser for yaml file"""
```

Comment:
use docstring instead of comments. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
fixed